wrapper.php

```diff
@@ -47,7 +47,9 @@
 
 			<?php
 			\Civil_First_Fleet\Component\newsroom_footer()->render();
-			\Civil_First_Fleet\Component\civil_footer()->render();
+			if ( ! wp_validate_boolean( get_option( 'newsroom-settings' )['branding']['civil_branding']['disable_footer'] ) ) {
```

Comment:
Same as above.

Reply:
I just realized that this is not using null coalescing. This would throw a notice and therefore is a blocker for me.